src/istgt_iscsi.c

```diff
@@ -6364,7 +6366,7 @@ int
 istgt_create_conn(ISTGT_Ptr istgt, PORTAL_Ptr portal, int sock, struct sockaddr *sa, socklen_t salen __attribute__((__unused__)))
 {
 	char buf[MAX_TMPBUF];
-	CONN_Ptr conn;
+	CONN_Ptr conn, existing_conn;
```

Comment:
existing_conn should be added for REPLICATION build. Let us move the declaration where it is being used.

Reply:
Done